enterprise/index.md

```diff
@@ -4,17 +4,137 @@ description: Information about Docker Enterprise Edition
+    compliance testing before they're reviewed and posted on Docker Store.
+    Apps are portable across different network and storage infrastructure, and
+    they work with new plugins without recoding.
+
+The Docker Certification Program broadens the Docker platform to a wide
+community of developers and IT ops teams at businesses of all sizes.
+Businesses that seek to modernize their applications can build and deploy on
+Docker EE for Linux and Windows, on any infrastructure. Many components of
+enterprise environments come from third parties, and the Docker Certified
+Program accelerates the adoption of these technologies into Docker
+environments with assurances and support.
+
+## Docker EE feature tiers
+
+Docker EE is available in three tiers:
+
+-  **Basic:** The Docker platform for certified infrastructure, with support
+   from Docker Inc. and certified containers and plugins from Docker Store.
+-  **Standard:** Adds advanced image and container management, LDAP/AD user
+   integration, and role-based access control. Together, these features
+   comprise *Docker Datacenter*.
+-  **Advanced:** Adds
+   [Docker Security Scanning](https://blog.docker.com/2016/05/docker-security-scanning/)
+   and continuous vulnerability monitoring.
+
+## Docker Enterprise Edition and Docker Community Edition release cycles
+
+Docker EE and Docker CE are released quarterly. Both use a time-based
+versioning scheme, so for example, Docker EE version 17.03 was released in
+March 2017.
+
+Each Docker EE release is supported and maintained for one year and receives
+security and critical bug fixes during this period.
+
+Each Docker CE quarterly release is maintained for 4 months, which gives
+Docker CE users a one-month window to update to the next version.
+
+Docker CE has two channels:
+
+-  **Edge:** Released monthly for users who want the latest and greatest
+   features.
+-  **Stable:** Released quarterly for users who want an easier-to-maintain
+   release cadence.
+
+The following diagram shows how Docker CE and Docker EE versions are released.
+
+![Docker CE and Docker EE lifecycle](images/docker-ce-and-ee-lifecycle.png)
+
+The Docker API version is independent of the Docker platform version. The API
+version doesn't change from Docker 1.13.1 to Docker 17.03. We maintain
+careful API backward compatibility and deprecate APIs and features slowly and
+conservatively. We remove features after deprecating them for a period of
+three stable releases. Docker 1.13 introduced improved interoperability
+between clients and servers using different API versions, including dynamic
+feature negotiation.
+
+The deterministic release train benefits the Docker project. Maintainers and
+partners who want to ship new features in Docker are guaranteed that new
+features are in the hands of Edge channel users within a month of being
+merged.
+
+## Upgrades and support
+
+Docker CE for Mac and Windows users get automatic upgrade notifications. If
```

Comment:
Again, since Mac and Docker for Windows (separate of EE engine for Windows Server) are currently CE only, it may be confusing to talk about them in this doc.